src/Service.Tests/dab-config.CosmosDb_NoSql.json

```diff
@@ -178,7 +186,7 @@
     },
     "TagAlias": {
       "source": {
-        "object": "graphqldb.tag"
+        "object": "graphqldb.planet"
```

Comment:
I don't understand why these changes are being made. The top-level entities are Character, Star, Tag, etc. and now you're setting the backing object to be planet for all of the entities? that seems incorrect and possibly breaking for many cosmos tests.

Can you elaborate more on what you're changing here within your PR description?

Reply:
Just put it in the PR description. This is existing functionality, but this config file was not representing it.

Reply:
I think we should actually modify existing test, to verify source.object value i.e. container name provided at the entity level overrides the global container. And that if none is provided at entity level, the global continues to be used.

Reply:
Per pr description:
> Updated source name in db-config for all the entities. Actually, all the entities belong to the same container i.e. planet. This was already happening in our test cases ref https://github.com/Azure/data-api-builder/blob/main/src/Service.Tests/CosmosTests/TestBase.cs#L135
I just updated the config file, to get it highlighted.

The file referenced shows that _containerName is a GUID, not `planet`:
```csharp
    internal string _containerName = Guid.NewGuid().ToString();
```

Reply:
Yes, I reverted it to GUID because, pipeline started failing, as these tests run parallelly, and it was trying to delete and create the same container again and again due to which tests were failing (only in pipelines) with "not found" error.
In this highlighted statement, I am saying that even override is happening in test base, still I have updated config to indicate, what is expected so that nobody should have to go through this logic, like I had to do to understand the behavior.